beluga/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ project(beluga VERSION 1.0)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+list(APPEND CMAKE_MODULE_PATH "/usr/local/lib/cmake/OpenVDB")
```

Comment:
@pvela2017 consider adding this to the `find_package` statement, as [PATHS](https://cmake.org/cmake/help/latest/command/find_package.html#full-signature). Same elsewhere.

Reply:
This was delayed.

Reply:
I _think_ we don't need this for OpenVDB as distributed by Ubuntu (and Debian).